src/notifications/generate.ts

```diff
@@ -148,6 +150,10 @@ export const notificationTitleMap: Record<
     `${ctx.user.name} just posted in ${ctx.source.name}. This post is waiting for your review before it gets published on the squad.`,
   user_gifted_plus: (ctx: NotificationGiftPlusContext) =>
     `Surprise! 🎁 ${ctx.gifter.username} thought of you and gifted you a one-year daily.dev Plus membership! How’s that for a thoughtful surprise?`,
+  user_received_award: (ctx: NotificationAwardContext) => {
+    const coreAmount = formatCoresCurrency(ctx.transaction.value);
+    return `You just received +${coreAmount} Cores from ${ctx.sender.username} as an Award! Keep creating great content!`;
```

Comment:
Does this not break when they award with free?

Reply:
🙈 
![image](https://github.com/user-attachments/assets/ca27f8d5-daf5-44de-9d3e-439009980c7d)

Reply:
AS-1086 will fix it.